changelog.md

```diff
@@ -4,23 +4,37 @@
 
 ### Features:
 
-- Added `--signer` flag to `message`, `list`, `map` and `single` scaffolding to allow customizing the name of the signer of the message
+- Added `starport generate dart` to generate a Dart client from protocol buffer files
+- Added `starport scaffold flutter` to scaffold a Flutter mobile app template
+
+## `v0.18.0`
+
+### Features:
+
+- Scaffold commands allow using custom types as fields
```

Comment:
I think we should precise "allows using previously scaffolded types" since not all custom types can be used